db/migrate/20240405154811_add_consultation_required_to_planning_application_constraint.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddConsultationRequiredToPlanningApplicationConstraint < ActiveRecord::Migration[7.1]
+  def change
+    add_column :planning_application_constraints, :consultation_required, :boolean, null: false, default: true
```

Comment:
Not really a problem now but this will cause a table lock as the default is backfilled - if we were live we'd use `change_column_null` and `change_column_default` after backfilling using `Modle.find_each`.

Wonder whether it's worth adding the [strong_migrations][1] gem to avoid this scenario going forward?

[1]: https://github.com/ankane/strong_migrations

Reply:
That's a good point — you've mentioned it a few times now and even if it's not strictly necessary yet it's probably better to start sooner than later. (Otherwise, chances are we won't get around to it until it's too late.)

I'll rewrite this migration using the pattern from the few recent migrations.